src/com/superiornetworks/icarus/commands/Command_smite.java

```diff
@@ -0,0 +1,66 @@
+package com.superiornetworks.icarus.commands;
+
+import net.pravian.bukkitlib.command.BukkitCommand;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import com.superiornetworks.icarus.ICM_Utils;
+
+public class Command_smite extends BukkitCommand 
+{
+  @Override
+  public boolean run(final CommandSender sender, Command cmd, String label, String[] args)
+  {
+    if (!TFM_AdminList.isSuperAdmin(sender, true))
+        {
+            sender.sendMessage(ICM_Utils.NO_PERMS_MESSAGE);
+            return true;
+        }
+    
+    int length = args.length;
+    
+    if (length == 0)
+    {
+      sender.sendMessage("/smite <player>");
+      return true;
+    }
+    
+    if (length == 1)
+    {
+      boolean playerFound = false;
```

Comment:
A boolean is not needed, test to see if the player is null, return true, and be done.